frontend/src/components/elements/BokehChart/BokehChart.tsx

```diff
@@ -35,6 +38,20 @@ class BokehChart extends React.PureComponent<Props> {
 
   private updateChart = (data: any): void => {
     const chart = document.getElementById(this.chartId)
+    const plot =
+      data && data.doc && data.doc.roots && data.doc.roots.references
+        ? data.doc.roots.references.find((e: any) => e.type === "Plot")
+        : undefined
+    if (plot) {
```

Comment:
Could you add a comment explaining the logic in the code block below?

Reply:
done